projects/plugins/boost/app/assets/src/js/pages/purchase-success/PurchaseSuccess.svelte

```diff
@@ -34,7 +39,21 @@
 				<ul class="jb-checklist my-2">
 					<li>{__( 'Automatic critical CSS regeneration', 'jetpack-boost' )}</li>
 					<li>{__( 'Performance scores are recalculated after each change', 'jetpack-boost' )}</li>
-					<li>{__( 'Dedicated email support', 'jetpack-boost' )}</li>
+
+					<li>
+						<!-- svelte-ignore missing-declaration Jetpack_Boost -->
+						{#if Jetpack_Boost.site.isAtomic}
+							<TemplatedString
+								template={__(
+									`Dedicated email support plus priority Live Chat if <link>your plan</link> <strong>includes Premium Support</strong>`,
```

Comment:
`<strong>` shouldn't include the word "includes". Only "Premium Support" should be bolded, as a feature name.

Reply:
Thanks! Fixed with 428fd51.